tests/test_restore_session_relaunches_codex_resume.py

```
@@ -0,0 +1,290 @@
+    app_path = Path(app_path_str)
+    if not app_path.exists():
+        print(f"SKIP: CMUX_APP_PATH does not exist: {app_path}")
+        return 0
+
+    cli_path = app_path / "Contents" / "Resources" / "bin" / "cmux"
+    if not cli_path.exists():
+        print(f"SKIP: bundled cmux CLI not found at {cli_path}")
+        return 0
+
+    bundle_id = _bundle_id(app_path)
+    socket_path = Path(f"/tmp/cmux-restore-session-codex-{bundle_id.replace('.', '-')}.sock")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-restore-session-codex-"

(S108)